srai/loaders/__init__.py

```diff
@@ -1,7 +1,8 @@
 """Loaders."""
 
 from .base import BaseLoader
+from .exceptions import LoadedDataIsEmptyException
 from .geoparquet_loader import GeoparquetLoader
 from .gtfs_loader import GTFSLoader
 
-__all__ = ["BaseLoader", "GeoparquetLoader", "GTFSLoader"]
+__all__ = ["BaseLoader", "GeoparquetLoader", "GTFSLoader", "LoadedDataIsEmptyException"]
```

Comment:
I would not add it here. I think importing an exception can come with `from srai.loaders.exceptions`, but feel free to change my mind ;)

Reply:
I moved it to srai.utils.exceptions. LMK if you think if it is sensible

Reply:
It's fine for me